source/docs/zero-to-robot/step-2/wpilib-setup.rst

```diff
@@ -1,280 +1,108 @@
 WPILib Installation Guide
-=============================
+=========================
 
-.. note:: This setup is for C++/Java teams. LabVIEW teams should proceed to :doc:`Installing LabVIEW for FRC </docs/zero-to-robot/step-2/labview-setup>`.
+This guide is intended for Java and C++ teams. LabVIEW teams can skip to :doc:`labview-setup`. Additionally, the below tutorial shows Windows 10, but the steps are identical for all OSes. Notes differentating OSes will be shown.
```

Comment:
I think the last 2 sentences are awkward.

I would not use an initialism for `OS`.